<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myPlayerQtClass</class>
 <widget class="QMainWindow" name="myPlayerQtClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的播放器</string>
  </property>
  <property name="windowIcon">
   <iconset resource="my_player_qt.qrc">
    <normaloff>:/my_player_qt/Resources/playerIcon.png</normaloff>:/my_player_qt/Resources/playerIcon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255);

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(0, 0, 0);
color:rgb(0, 0, 0)</string>
   </property>
   <widget class="videoWidget" name="openGL_widget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0)</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="videoInfoWidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>210</width>
      <height>450</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(89,87,87);
border:1px</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(89,87,87);
border-color:rgb(35, 35, 35)</string>
     </property>
     <attribute name="title">
      <string>播放列表</string>
     </attribute>
     <widget class="QListWidget" name="videoListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>440</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>400</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true">background-color:rgb(35, 35, 35);</string>
     </property>
     <attribute name="title">
      <string>视频信息</string>
     </attribute>
     <widget class="QTextEdit" name="videoInfo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>200</width>
        <height>440</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="loadingLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>160</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iconBackground">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/my_player_qt/Resources/playerIcon.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border: 0px</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="nextButton">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>22</y>
       <width>44</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/my_player_qt/Resources/CIRCLE_SKIP_NEXT.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="forwardButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>22</y>
       <width>44</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/my_player_qt/Resources/CIRCLE_FAST_FORWARD.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover{border-image:url(:/my_player_qt/Resources/CIRCLE_PLAY.png);}
QPushButton:hover{border-image: url(:/my_player_qt/Resources/CIRCLE_PLAY.png);}	</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="rewinButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>22</y>
       <width>44</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/my_player_qt/Resources/CIRCLE_REWIND.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="previousButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>22</y>
       <width>44</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/my_player_qt/Resources/CIRCLE_SKIP_PREVIOUS.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="slider" name="playSlider">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>600</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="curTime">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;00:00:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="totalTime">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;00:00:00&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSlider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="volumeButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/my_player_qt/Resources/VOLUME.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="textLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 14px;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string>正在加载...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openLocalFile"/>
    <addaction name="openWebFile"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="previousVideo"/>
    <addaction name="nextVideo"/>
    <addaction name="volumeMute"/>
    <addaction name="pauseAction"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="openVideoList"/>
    <addaction name="fullScreenAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openLocalFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开本地视频</string>
   </property>
  </action>
  <action name="openWebFile">
   <property name="text">
    <string>打开网络视频</string>
   </property>
  </action>
  <action name="openVideoList">
   <property name="text">
    <string>播放列表</string>
   </property>
   <property name="toolTip">
    <string>播放列表</string>
   </property>
  </action>
  <action name="pauseAction">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="fullScreenAction">
   <property name="text">
    <string>全屏</string>
   </property>
  </action>
  <action name="volumeMute">
   <property name="text">
    <string>静音</string>
   </property>
  </action>
  <action name="nextVideo">
   <property name="text">
    <string>下一个视频</string>
   </property>
  </action>
  <action name="previousVideo">
   <property name="text">
    <string>上一个视频</string>
   </property>
  </action>
  <action name="screenshotAction">
   <property name="text">
    <string>截图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>slider</class>
   <extends>QSlider</extends>
   <header>slider.h</header>
  </customwidget>
  <customwidget>
   <class>videoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videoWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="my_player_qt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>sliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>445</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>sliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoListWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>clickedVideoList(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeButton</sender>
   <signal>clicked()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>setMute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>setVolume()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousButton</sender>
   <signal>clicked()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>skipPrevious()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>skipNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pauseAction</sender>
   <signal>triggered()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullScreenAction</sender>
   <signal>triggered()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>fullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openWebFile</sender>
   <signal>triggered()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>netAddressInput()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openLocalFile</sender>
   <signal>triggered()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>volumeMute</sender>
   <signal>triggered()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>setMute()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>openVideoList</sender>
   <signal>triggered()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>showVideoList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousVideo</sender>
   <signal>triggered()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>skipPrevious()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextVideo</sender>
   <signal>triggered()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>skipNext()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rewinButton</sender>
   <signal>clicked()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>seekBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>448</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>forwardButton</sender>
   <signal>clicked()</signal>
   <receiver>myPlayerQtClass</receiver>
   <slot>seekForward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open()</slot>
  <slot>sliderPressed()</slot>
  <slot>sliderRelease()</slot>
  <slot>play()</slot>
  <slot>setVolume()</slot>
  <slot>clickedVideoList(QListWidgetItem*)</slot>
  <slot>setMute()</slot>
  <slot>skipPrevious()</slot>
  <slot>skipNext()</slot>
  <slot>fullScreen()</slot>
  <slot>showVideoList()</slot>
  <slot>netAddressInput()</slot>
  <slot>seekBack()</slot>
  <slot>seekForward()</slot>
 </slots>
</ui>
